lib/sles4sap_publiccloud.pm

```diff
@@ -201,6 +201,7 @@ sub sles4sap_cleanup {
 
 sub get_hana_topology {
     my ($self) = @_;
+    $self->run_cmd(cmd => 'cs_wait_for_idle -s 5');
```

Comment:
Please add a timeout. On prem tests and `sles4sap/publiccloud/hana_sr_takeover` are using 120 seconds. See: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/publiccloud/hana_sr_takeover.pm#L40